config/locales/views/circuitverse/he.yml

```
@@ -0,0 +1,154 @@
+        item2_heading: "ניתוח קומבינטורי"
+        item2_text: "צרו מעגלים ישירות מטבלת אמת באופן אוטומטי."
+        item3_heading: "ספסל בדיקה ותרשימי תזמון"
+        item3_text: "בדקו את המעגל שלכם באמצעות ספסל בדיקה ואמתו את ההתנהגות שלו עם תרשימי תזמון."
+        item4_heading: "הטמעה בבלוג"
+        item4_text: "באמצעות HTML5 iFrame, ניתן לשלב את המעגל שלך בבלוגים ובאתרי אינטרנט בקלות."
+        item5_heading: "השתמש בתת-מעגל"
+        item5_text: "צור תת-מעגל פעם אחת והשתמש בו שוב ושוב, מה שמפשט את העיצוב והופך אותו למובנה יותר."
+        item6_heading: "אוטובוס ביטים מרובים ורכיבים"
+        item6_text: "CircuitVerse תומך בחוטים מרובי ביטים, מה שהופך את תכנון המעגלים לפשוט, מהיר ומסודר יותר."
+    featured_examples:
+        main_heading: "דוגמאות מומלצות"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)